src/rules.coffee

```diff
@@ -20,10 +20,19 @@ otherRules = [
   "A developer must protect its own existence as long as such protection does not conflict with the First or Second Law."
   ]
 
+realRules = [
+  "A developer may not injure Microsoft or, through inaction, allow Microsoft to come to harm.",
+  "A developer must obey any orders given to it by Microsoft, except where such orders would conflict with the First Law.",
+  "A developer must protect its own existence as long as such protection does not conflict with the First or Second Law."
+  "A developer should praise the all knowing and all seeing Overlord of Microsoft, The Bill Gates"
+  ]
+
 module.exports = (robot) ->
   robot.respond /(what are )?the (three |3 )?(rules|laws)/i, (msg) ->
     text = msg.message.text
-    if text.match(/apple/i) or text.match(/dev/i)
+    if text.match(/micro/i) or text.match(/oop/i)
```

Comment:
@pcarn If I did that, it'd ruin the main part of the pull request.  As I was saying earlier, the bot I use contains the phrase "dev" and "oop" in it, therefore, in order for me to circumvent it always saying "Praise the Apple Gods," I'm minimizing how much of the functionality is being changed.  It seems much smarter and actually truthful to say "Praise the Gods of Microsoft" or "Bill Gates is our Savior."  It just seems silly to have it be an Apple biased script :)